src/content/special-interest-groups/animal-genomics/meetings.md

```diff
@@ -11,9 +11,17 @@ of every month at 4 PM CET / 10 AM EDT / 7 AM PDT for 1h.
 
 The first meeting will be on Wednesday, April 17th, at 16:00 CEST. For this inaugural session, we are honored to announce Christa Kühn, president of the Friedrich Loeffler Institute, BovReg project coordinator and former coordinator of EuroFAANG RI, as the speaker. Christa is well-known in the Bovine field for her important contributions to animal genetics. Her talk will be followed by a round table where the structure of the series will be announced and discussed with the participants.
```

Comment:
Maybe we should update this then to be a general introductory text for all the meetings?